### DISCUSSION TITLE
How to use `io_uring_prep_socket_direct` with `bind` & `listen`?

### DISCUSSION THREAD

**Discussion by YoSTEALTH** [axboe/liburing#1092] [Q&A]
How to use `io_uring_prep_socket_direct` with `bind` & `listen`?

**Comment by axboe** [comment]
You can't, as there's no bind/listen io_uring command. You'd have to first install it as a normal file descriptor first, if you have a direct one and need to use regular syscalls with it. See ```io_uring_prep_fixed_fd_install.3```.
